PDF/src/PDF.Server/Controllers/PDFController.cs

```diff
@@ -67,14 +67,9 @@ private string getFromHeaders(string key)
 
             // execute the Node.js component
             result = await nodeServices.InvokeAsync<JSONResponse>("./pdf", "schoolbus_permit", rawdata, options); 
-            
-            HttpContext.Response.ContentType = "application/pdf";
-            string filename = @"output.pdf";
-            HttpContext.Response.Headers.Add("x-filename", filename);
```

Comment:
could filename be a string representing the permit number?
might be useful/convenient for the user to have the 'save as' name preloaded